terraform/modules/paas/selfservice.tf

```diff
@@ -50,3 +50,15 @@ resource "cloudfoundry_network_policy" "selfservice" {
     port            = "8080"
   }
 }
+
+module "selfservice_credentials" {
+  source = "../credentials"
+  pay_low_pass_secrets = lookup(local.selfservice_credentials, "pay_low_pass_secrets")
```

Comment:
local.selfservice_credentials needs to be defined at the top. See: https://github.com/alphagov/pay-omnibus/blob/develop/terraform/modules/paas/directdebit-connector.tf#L1

Reply:
fixed